lib/config/config-initializer.js

```diff
@@ -417,16 +429,62 @@ function askInstallModules(modules, packageJsonExists) {
 function promptUser() {
 
     return inquirer.prompt([
+        {
+            type: "list",
+            name: "purpose",
+            message: "How would you like to use ESLint?",
+            default: "problems",
+            choices: [
+                { name: "To check syntax only", value: "syntax" },
+                { name: "To check syntax and find problems", value: "problems" },
+                { name: "To check syntax, find problems, and enforce code style", value: "style" }
+            ]
+        },
+        {
+            type: "list",
+            name: "moduleType",
+            message: "What type of modules does your project use?",
+            default: "esm",
+            choices: [
+                { name: "JavaScript modules", value: "esm" },
+                { name: "CommonJS", value: "commonjs" },
+                { name: "None of these", value: "none" }
+            ]
+        },
+        {
+            type: "list",
+            name: "framework",
+            message: "Which framework does your project use?",
```

Comment:
It's a minor thing, but this wording could be interpreted to mean that all projects should be using some framework (either React, Vue, or some other framework not listed here). I assume this isn't the intended interpretation, since not all projects use frameworks. What would you think about rewording this question to avoid that potential interpretation? For example:

> Are you using either of the following frameworks?

Reply:
I think the third option, "None of these", makes this question clear. I'm not a fan of changing to a yes/no question and then not asking for a yes/no response.